### DISCUSSION TITLE
Support F20–F24

### DISCUSSION THREAD

**Discussion by smammy** [rxhanson/Rectangle#1232] [Ideas]
macOS supports fkeys up to F24, but Rectangle only supports F1–F19. (I remap certain fn+key combinations to high-numbered fkeys as a workaround for #288.)

**Comment by rxhanson** [comment]
Thanks for reporting. As mentioned in that linked issue, since I have ownership of that MASShortcut fork, I'll see what I can do to allow up to F24.

**Reply by smammy** [reply]
Well, here's F20 support, for what it's worth: rxhanson/MASShortcut#1

**Reply by smammy** [reply]
Although apparently F21–F24 are cursed.
